tools/cli/pkg/command/runtime.go

```diff
@@ -156,6 +157,9 @@ func (cmd *RuntimeCommand) Validate() error {
 	if cmd.opDetail {
 		cmd.params.OperationDetail = runtime.AllOperation
 	}
+	if cmd.params.IncludeDeleted == true && len(cmd.params.InstanceIDs) == 0 {
+		return fmt.Errorf("need to provide some Instance IDs when using --include-deleted")
```

Comment:
unfortunate but necessary limitation, in order to efficiently reconstruct partial instances, KEB needs KCP to provide Instance IDs and only those specifically because that is the FK in the operations table and events table.